nerfstudio/data/datamanagers/base_datamanager.py

```diff
@@ -335,6 +339,16 @@ class VanillaDataManagerConfig(DataManagerConfig):
     """
     patch_size: int = 1
     """Size of patch to sample from. If > 1, patch-based sampling will be used."""
+    prefetch_factor: int = 4 # increasing prefetch_factor was not beneficial
+    """The limit number of batches a worker will start loading once an iterator is created. 
+    More details are described here: https://pytorch.org/docs/stable/data.html#torch.utils.data.DataLoader"""
+    dataloader_num_workers: int = 1
+    """The number of workers performing the dataloading from either disk/RAM, which 
+    includes collating, pixel sampling, unprojecting, ray generation etc."""
+    use_ray_train_dataloader: bool = True
+    """Allows parallelization of the dataloading process with multiple workers."""
+    cache_binaries: bool = True
+    """If True, caches the images as binary strings to RAM"""
```

Comment:
in python 3.x there's `bytes` type that helps differentiate versus in python 2.x one had to have "binary data in strings"

so maybe just "When enabled, cache raw image files to RAM"